sirepo/job_api.py

```diff
@@ -95,6 +95,36 @@ def api_downloadDataFile(simulation_type, simulation_id, model, frame, suffix=No
         )
 
 
+@api_perm.require_user
+def api_exportRSOptConfig(simulation_type, simulation_id, filename):
+    import zipfile
+    t = sirepo.template.import_module(simulation_type)
+    assert hasattr(t, 'export_rsopt_config'), 'Export rsopt unavailable'
+    e = t.export_rsopt_config(
+        simulation_db.read_simulation_json(simulation_type, sid=simulation_id),
+        filename
+    )
+    with simulation_db.tmp_dir() as tmp:
+        fz = tmp.join(filename)
```

Comment:
`sirepo.util.safe_path` instead of `join`. The filename could be malicious. 

One better option to consider - Use a non-user supplied filename and then only return the user supplied filename in the final `return http_reply.gen_file_as_attachment()`. I don't really know how zipfile and gen_file_as_attachment work so maybe this won't work.